test/integration/servicebus/queue.test.js

```diff
@@ -18,9 +18,10 @@ describe('ServiceBus', function() {
       test.client = new AMQPClient(Policy.ServiceBusQueue);
     });
 
-    afterEach(function() {
+    afterEach(function(done) {
       return test.client.disconnect().then(function() {
         test.client = undefined;
+        done();
```

Comment:
hmm this shouldn't be needed, were you experiencing issues where it didn't complete this step in time?